tests/unit_tests/scheduler/test_lsf_driver.py

```diff
@@ -736,3 +742,35 @@ async def mock_submit(*args, **kwargs):
     assert "LSF kill failed" not in caplog.text
     assert "LSF kill failed" not in capsys.readouterr().err
     assert "LSF kill failed" not in capsys.readouterr().out
+
+
+@pytest.mark.parametrize(
+    "job_submitted_timestamp, submitted_before, expected_result",
+    [
+        pytest.param(
+            time.time() - 1.0,
+            time.time(),
+            set(["1"]),
+            id="job_submitted_before_deadline",
+        ),
+        pytest.param(
+            time.time() + 1.0, time.time(), set(), id="job_submitted_after_deadfline"
```

Comment:
deadfline -> deadline

Reply:
can you add a test to nail down the expected beviaviour when timestamp and deadline is equal?